optics-sop/src/Generics/SOP/Optics.hs

```diff
@@ -21,49 +21,49 @@ import Generics.SOP
 import Optics hiding (to)
 
 -- | Iso between a generic type and its representation.
-rep :: (Generic a, Generic b) => Iso i a b (Rep a) (Rep b)
+rep :: (Generic a, Generic b) => Iso a b (Rep a) (Rep b)
 rep = iso from to
 
--- | Iso induced by the 'SOP' newtype.
-sop :: Iso i (SOP f xss) (SOP g yss) (NS (NP f) xss) (NS (NP g) yss)
+-- | Isonduced by the 'SOP' newtype.
```

Comment:
It seems you search-and-replaced a bit too much here ;) Same below.

Reply:
Thanks, fixed